tycho-core/src/test/java/org/eclipse/tycho/p2resolver/TargetDefinitionFileTest.java

```diff
@@ -44,7 +43,7 @@ public void testTarget() throws Exception {
 
         InstallableUnitLocation location = (InstallableUnitLocation) locations.get(0);
         assertEquals(1, location.getRepositories().size());
-        assertEquals(URI.create("https://download.eclipse.org/eclipse/updates/3.5/"),
+        assertEquals("https://download.eclipse.org/eclipse/updates/3.5/",
```

Comment:
Are the changes in this file necessary? Why using sting instead of URI for equals?

Reply:
Yes it is necessary because I had to change `Repository.getLocation`  return type from `URI` to `String`.
Since URI will never be .equal() to any object of type String I had to change the expected value to also be String